rainforest/reporter.go

```diff
@@ -68,31 +69,13 @@ func (c *Client) GetRunDetails(runID int) (*RunDetails, error) {
 		return &runDetails, err
 	}
 
-	// NOTE: This extra request is only necessary because `updated_at` is not
-	// currently exposed in the `/runs/:id` endpoint. This may change in the future:
-	// https://github.com/rainforestapp/rainforest-cli/issues/216
-	var runTests []RunTestDetails
-	url = "runs/" + strconv.Itoa(runID) + "/tests?page_size=" + strconv.Itoa(runDetails.TotalTests)
```

Comment:
This endpoint is paginated with a max page size of 100. Therefore, if there were >100 run tests, you would not get a complete set of results in your JUnit file. However, we don't need to make this request to retrieve `updated_at` anymore - this information is already present in the previous request above. And that request is NOT paginated, so you will get complete results just by removing this.